packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -1514,7 +1514,11 @@ describeCompat("stashed ops", "NoCompat", (getTestObjectProvider, apis) => {
 		},
 	);
 
-	it("offline blob upload", async function () {
+	it("blob upload before loading", async function () {
+		// TODO: https://dev.azure.com/fluidframework/internal/_workitems/edit/19035
```

Comment:
Better to include a brief description and not the full link, since this link is not publicly usable.

```suggestion
		// TODO: AB#19035: Blob upload from an offline state does not currently work before establishing a connection on ODSP due to epoch mismatch.
```

(feel free to adjust the description)